submissions/description/insufficient_security_configurability/verification_of_contact_method_not_required/template.md

```diff
@@ -0,0 +1,21 @@
+# Lack of Verification Email
```

Comment:
Incorrect title, needs to be updated to "Verification of contact method not required"